spirv_reflect.c

```diff
@@ -386,6 +390,11 @@ static SpvReflectResult ReadU32(
   return result;
 }
 
+#define UNCHECKED_READU32(parser, word_offset, value)             \
```

Comment:
Is this just for a a throw away read?

Reply:
No, not a throwaway read. It is a true read whose value is truly used. It is just does not have the check code so that it can be performed in other contexts besides the initial parsing context.